Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ukhomeofficedigital/centos-base:v0.5.0
```

Comment:
Were there any specific differences between v0.5.0 and what's currently latest? Might have been a reason for restricting this to a specific version.

Reply:
I'm not entirely sure, v0.5.0 didn't work which is why I changed it in the first place. The latest version is v0.5.5 though.